components/scream/src/dynamics/homme/dynamics_driven_grids_manager.cpp

```diff
@@ -139,14 +131,17 @@ build_grids (const std::set<std::string>& grid_names,
 
 void DynamicsDrivenGridsManager::build_dynamics_grid () {
   if (m_grids.find("Dynamics")==m_grids.end()) {
+    // Build the Physics GLL grid, to be used as 'unique grid' in the dyn grid
+    build_physics_grid("Physics GLL");
```

Comment:
Just a question on syntax, while I'm looking at this code.  We have "Physics GLL" and "Physics" used throughout the code.  Would it make sense to never use "Physics" and require the user to be specific what kind of physics grid is in use?  So have "Physics GLL", "Physics Point", and eventually "Physics PG"

Reply:
Sure, we can do that. I can remove any use of `Physics`, and maybe call the non-GLL phys grid `Physics FV` rather than `PG`, They are all phys grids, after all, we are distinguishing them by the numerical algorithm they are used with.

Reply:
Actually, there are two uses of 'Physics' in scorpio in/out, which can be purged (we now can do I/O on dyn grid too). Other than those, the places that have `Physics` as a grid name are either tests building a PointGrid (and they give it the name "Physics"), or the PhysicsOnlyGridManager, which one could argue it should be called "PointGridOnlyGridManager" maybe.

I'm gonna move the PhysicsOnlyGridManager to share/grid, and use it for io tests too.